src/network/taraxa_capability.hpp

```diff
@@ -146,6 +156,7 @@ struct TaraxaCapability : virtual CapabilityFace {
   util::ThreadPool syncing_tp_{1, false};
   SyncingState syncing_state_;
   PeersState peers_state_;
+  mutable std::shared_mutex sync_access_;
```

Comment:
I would not know what is this mutex guarding without some checking. On the other hand if you group it with sync_queue_ (just move it's declaration next to it and maybe give some empty new line around) it would be clear straight away